src/tests/JIT/opt/Compares/conditionalInverts.csproj

```diff
@@ -1,4 +1,8 @@
 <Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <!-- Needed for CLRTestEnvironmentVariable -->
+    <RequiresProcessIsolation>true</RequiresProcessIsolation>
```

Comment:
I am not sure whether this is the best way to address the test failures.

Reply:
I think you should remove this and `<OutputType>Exe</OutputType>` below... I don't recall what the guidance ended up being on `OutputType`, cc @markples

Reply:
Correct - remove the `<OutputType>Exe</OutputType>`.  In addition, we no longer use `Main` methods (automated conversion renamed them to `TestEntryPoint`, but the name doesn't matter).  Mark that method with `[Fact]` (and add `import Xunit;`).

However, these tests with `CLRTestEnvironmentVariable` do need the `<RequiresProcessIsolation>true</RequiresProcessIsolation>` line that you originally had.  A "normal" test does not need this.

Reply:
I'm curious, why didn't this issue (needing to add process isolation) occur for the tests in the `conditionalIncrements.cs` ?

Reply:
I will be improving the error check around the msbuild files and adding some documentation to better explain this.  Tests in many directories (including JIT/opt) are now running in a new "merged tests" mode.  They still build as individual projects but are invoked from a wrapper that calls many tests (to save on process startup).  Only [Fact] (or other Xunit-style attributes) methods are executed, so a test with a Main would actually be silently dropped today (something I definitely want to fix).

RequiresProcessIsolation is required when the test has some process-level setting that isn't really compatible with the new merged test system.  However, it still goes through the [Fact] methods for consistency (and to help simplify our github/azure devops test scripts).  Without it, those environment variables wouldn't be set because of how the scripts are invoked.